libraries/textcomposer/src/main/kotlin/io/element/android/libraries/textcomposer/TextComposer.kt

```diff
@@ -96,7 +96,7 @@ fun TextComposer(
     focusRequester: FocusRequester = FocusRequester(),
     onSendMessage: (String) -> Unit = {},
     onResetComposerMode: () -> Unit = {},
-    onComposerTextChange: (CharSequence) -> Unit = {},
+    onComposerTextChange: (String) -> Unit = {},
```

Comment:
The lambda `BasicTextField.onValueChange` parameter is a String, so this change has no impact.